main.go

```diff
@@ -161,16 +163,20 @@ func main() {
 		setupLog.Error(err, "failed to create discovery client")
 		os.Exit(1)
 	}
-
+	webhookServer := webhook.NewServer(webhook.Options{
+		Port: 9443,
+	})
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		Cache:                  cache.Options{DefaultNamespaces: map[string]cache.Config{namespace: {}}},
+		WebhookServer:          webhookServer,
```

Comment:
why do we need this webhook server?

Reply:
Yeah, I have the very same question - we didn't have any webhook configuration before (at least I don't see i there) and we don't use OLM's way of deploying admission webhooks, so do we really need it?

Reply:
we had port before which was deprecated and now removed.
from the documentation:
```
        // Port is the port that the webhook server serves at.
	// It is used to set webhook.Server.Port if WebhookServer is not set.
	//
	// Deprecated: Use WebhookServer instead. A WebhookServer can be created via webhook.NewServer.
	Port int
```

for more details I've linked the PR removing deprecated manager options field in the description.

Reply:
I'm curious if the default value is the same as we set here (port 9443)?

Reply:
yes the default port value is 9443, good thinking/curiosity. 
From the following doc - https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/manager/manager.go#L244-L247 - Manager will create a default webhook server with default setting which works for us. 
Let me remove the webhook server then.

Reply:
removed in [30e011b](https://github.com/codeready-toolchain/member-operator/pull/615/commits/30e011bdb66ae8242ec4c6290a805862185a4920)